reana_client/cli/workflow.py

```diff
@@ -729,22 +732,87 @@ def workflow_logs(ctx, workflow, access_token, json_format, steps=None):  # noqa
     """
     from reana_client.api.client import get_workflow_logs
 
+    available_filters = {
+        "step": "job_name",
+        "compute_backend": "compute_backend",
+        "docker_img": "docker_img",
+        "status": "status",
+    }
+    compute_backends = {
+        "kubernetes": "Kubernetes",
+        "htcondor": "HTCondor",
+        "slurm": "Slurm",
+    }
+    steps = []
+    chosen_filters = dict()
+
     logging.debug("command: {}".format(ctx.command_path.replace(" ", ".")))
     for p in ctx.params:
         logging.debug("{param}: {value}".format(param=p, value=ctx.params[p]))
     if workflow:
+        if filters:
+            try:
+                for f in filters:
+                    key, value = f.split("=")
+                    if key not in available_filters:
+                        click.echo(
+                            click.style(
+                                "Error: filter '{}' is not valid\nAvailable filters are {}".format(
```

Comment:
Still some room for prettification:

```
Available filters are ['step', 'compute_backend', 'docker_img', 'status']
```

could be said:

```
Available filters are 'compute_backend', 'docker_img', 'status', 'step'.
```

Just join the list items into a string instead of printing "naked" Python list.

Reply:
Fixed!